### ISSUE TITLE
Python 3 Compatibility

### ISSUE THREAD

**Issue by 183amir** [idiap/bob#217]
I am opening this issue to keep track of Python 3 compatibility issues.

**Comment by 183amir** [comment]
https://github.com/bioidiap/bob.ip.flandmark/issues/5

**Comment by tiagofrepereira2012** [comment]
Hey Amir, we have tests against Python 3 on Travis
https://travis-ci.org/bioidiap/

This is not enough?

Cheers

**Comment by 183amir** [comment]
No. Some tests are just skipped when they are done in python 3. Like the
issue I referenced here.
